htdocs/core/lib/accounting.lib.php

```diff
@@ -73,6 +73,38 @@ function accounting_prepare_head(AccountingAccount $object)
 	return $head;
 }
 
+/**
+ *	Prepare array with list of tabs for accounting transaction
+ *
+ *	@param	BookKeeping	$object		Bookkeeping
+ *	@param	string		$mode		Mode _tmp if operation are drafted
+ *	@param	string		$type		Type of list "sub" (for subsidiary list) or not
+ *	@param	string		$backtopage Back to page (return on ledger by default)
+ *	@return	array<array{0:string,1:string,2:string}>	Array of tabs to show
+ */
+function accounting_transaction_prepare_head(BookKeeping $object, $mode = '', $type = '', $backtopage = '%2Faccountancy%2Fbookkeeping%2Flistbyaccount.php')
+{
```

Comment:
@aspangaro no need to urlencode because it will be done after

Reply:
Thanks @frederic34. PR done